pkgs/development/python-modules/pyhocon/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+# Runtime inputs:
+, pyparsing
+# Check inputs:
+, setuptools
+, pytest
+, mock
+}:
+
+buildPythonPackage rec{
+  pname = "pyhocon";
+  version = "0.3.51";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "10l014br012fa583rnj3wqf6g9gmljamcwpw4snqwwg15i0dmkll";
+  };
+
+  propagatedBuildInputs = [ pyparsing ];
+
+  checkInputs = [ setuptools pytest mock ];
+
+  # Tests fail because necessary data files aren't packaged for PyPi yet.
+  # See https://github.com/chimpler/pyhocon/pull/203
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = https://github.com/chimpler/pyhocon/;
+    description = "HOCON parser for Python";
+    # Long description copied from <GitHub>/setup.py on 2019-03-06.
```

Comment:
```suggestion
```

Reply:
I'd rather keep this. Only slightly worse than no long description at all is a copy/pasted description with no reference cited. :) However, I don't like my shorthand, so I'll be more explicit.